src/extension/subscriptions.ts

```diff
@@ -199,6 +216,16 @@ export const setupSubscriptions = (
     vscode.window.registerWebviewViewProvider('malloyHelp', provider)
   );
 
+  context.subscriptions.push(
+    vscode.languages.registerCodeLensProvider(
+      {
+        language: 'malloy-sql',
+        scheme: 'file',
```

Comment:
This limit where this works to the desktop app, because the web app uses something like `vscode-vfs:` for there file scheme.